ClothesApp/ClothesApp/Program.cs

```diff
@@ -1 +1,140 @@
-﻿Console.WriteLine("Hello");
\ No newline at end of file
+﻿using ClothesApp;
+using ClothesApp.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Internal;
+
+await using var db = new ClothesAppContext();
+
+var productsByBrandIdLazy = await GetProductsByBrandIdLazy(db, 1);
```

Comment:
Getting this variable doesn't show lazy loading, you just call db once and don't use navigation properties later